### ISSUE TITLE
STS: reproduce earlier AUC = 0.85 predicting mortality from pre-op ECG

### ISSUE THREAD

**Issue by erikr** [broadinstitute/ml4h#271]
Goal is to learn post-operative mortality for cardiac surgical patients at MGH from the most recent pre-operative ECG.

Recent results are significantly worse than our initial results: https://github.com/broadinstitute/ml/issues/202#issuecomment-610528628

**ROC for mortality: ECGs resampled to 2500 samples**
```
./scripts/tf.sh \
    /$HOME/ml/ml4cvd/recipes.py \
    --mode train \
    --logging_level INFO \
    --tensors /data/partners_ecg/mgh/hd5 \
    --input_tensors \
        ecg_2500_sts \
    --output_tensors \
        sts_death \
    --sample_csv /data/sts/mgh-all-features-labels.csv \
    --inspect_model \
    --optimizer adam \
    --epochs 100 \
    --batch_size 32 \
    --learning_rate 0.0002 \
    --conv_x 71 \
    --output_folder $HOME \
    --id sts-ecg-death-all-resampled-to-2500
```
![per_class_roc_sts_death-2500-downsampled](https://user-images.githubusercontent.com/1253422/82405825-f8b6e700-9a32-11ea-9d32-0fb76a093166.png)
![metric_history_sts-ecg-death-all-downsampled-to-2500](https://user-images.githubusercontent.com/1253422/82405923-3451b100-9a33-11ea-86e7-a18e25ec851a.png)

**ROC for mortality: ECGs resampled to 5000 samples**
```
./scripts/tf.sh \
    /$HOME/ml/ml4cvd/recipes.py \
    --mode train \
    --logging_level INFO \
    --tensors /data/partners_ecg/mgh/hd5 \
    --input_tensors \
        ecg_5000_sts \
    --output_tensors \
        sts_death \
    --sample_csv /data/sts/mgh-all-features-labels.csv \
    --inspect_model \
    --optimizer adam \
    --epochs 100 \
    --batch_size 32 \
    --learning_rate 0.0002 \
    --conv_x 71 \
    --output_folder $HOME \
    --id sts-ecg-death-all-resampled-to-5000
```
![per_class_roc_sts_death](https://user-images.githubusercontent.com/1253422/82405844-02404f00-9a33-11ea-90ca-876d887de641.png)
![metric_history_sts-ecg-death-all-resampled-to-5000](https://user-images.githubusercontent.com/1253422/82405934-3b78bf00-9a33-11ea-88c2-6fc1a6b0f274.png)
However, we do not know some vital information:
1. label prevalence in 2500- vs 5000-sample ECGs (would be addressed by issue https://github.com/broadinstitute/ml/issues/270)
2. label prevalence of any given experiment; technically info is in the log file aggregated stats but requires some digging (would be addressed by issue https://github.com/broadinstitute/ml/issues/266).
